### ISSUE TITLE
Farfetch.com: Discrepancy in _abck Cookie Domain Value Between PS-Analysis-Tool, Network Requests Tab and DevTools Application Cookies' Tab

### ISSUE THREAD

**Issue by jlacher** [GoogleChromeLabs/ps-analysis-tool#138]
**Describe the bug**

**To Reproduce**
1. Go to [](https://www.farfetch.com/)
2. Go to the Privacy Sandbox tab on DevTools
3. Open the Cookies section on left and select https://www.farfetch.com/
4. Search for the _abck cookie and look for the domain value under the domain column
5. Observe that the domain is farfetch.net
6. Go to the network requests tab on DevTools
7. Reload the page and search for farfetch.net
8. Observe The Headers section of one of the requests 
9. Look for a Set-Cookie for '_abck' and observe it's domain is set for farfetch.net

**Expected behavior**
The domain as it appears in the Application tab in the Chrome DevTools should be consistent with ps-analysis-tool and the network requests tab

**Screenshots**

On the ps-analysis-tool:

<img width="567" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/127773307/94bd695d-2f8a-400f-81b9-9fcc7fdbe7d9">

It can be observed that the domain for _abck is farfetch.net

On the network requests tab:
<img width="996" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/127773307/2825bc90-c0a3-4825-9abe-a8cb95fcb5ef">

It can be observed that the domain for _abck is farfetch.net

On Chrome DevTools' Application tab:
<img width="562" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/127773307/95be44a6-ec24-4608-80cc-29e56bf96949">

It can be observed that the domain for _abck is farfetch.com

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome Canary
 - Version 118

**Additional context**

**Comment by gagan0123** [comment]
@jlacher 

Thank you for reporting this issue.

We already know there are some discrepancies with the way this extension handles partitioned cookies.

Our team is currently exploring some low-level APIs to prevent this kind of mismatch in the data displayed by this extension and the Chrome DevTools. We're also examining a similar issue here: https://github.com/GoogleChromeLabs/ps-analysis-tool/issues/85

We will keep you informed of any progress made on this issue.

**Comment by sayedtaqui** [comment]
@jlacher

This was happening because there were two requests trying to set cookies with the same name but different domains. When we looked closely, we found that the later request was overriding the previous cookie (see screencast)

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/e2c9ecb0-6949-4b97-a522-037c62ecf2b6


Therefore, we should avoid the override and list both the cookies. It was implemented and fixed in https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/140 and should be available in the next release. I noticed that the default developer tool also shows both cookies occasionally.

![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/e0c08e56-59f9-4ee8-bdee-ce2f24f564ba)
